frontend/src/App.js

```diff
@@ -64,12 +64,24 @@ class App extends Component {
 
   }
   render() {
+    if (this.props.view === constants.MAP_VIEW) {
```

Comment:
Might as well put this into a function instead of calculating it in render. Think `renderMainView() `